docs/pages/docs/apis/auth.mdx

```diff
@@ -195,6 +195,31 @@ const { withAuth } = createAuth({
 });
 ```
 
+#### Bootstrapping a default user for local development at seed stage
+
+If you frequently reset your database during development and get frustrated having to create a user each time you want to log in, you can bootstrap a user in your seeding script.
+
+```typescript
+// seed-admin-user.ts
+import { KeystoneContext } from '@keystone-6/core/types';
+
+export default async (context: KeystoneContext) => {
+  // For local development convenience bootstrap the first admin user
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
Rather than environment, we tend to recommend `process.argv`, as seen in https://github.com/keystonejs/keystone/blob/44d94482770bdfbaf1be2e224912df854f7ee934/examples/blog/keystone.ts

```suggestion
  if (process.argv.includes('--seed-data')) {
```